### ISSUE TITLE
Applying for the plot code of the Figure 4

### ISSUE THREAD

**Issue by smellYang** [sayakpaul/Training-BatchNorm-and-Only-BatchNorm#1]
Hi, thanks for the nice work! I am very interested in the figure results of the distribution of \gamma, i.e. Figure 4 described in the paper, but I am confused about how to plot these figures. So could you provide me with some reference codes to plot these figures?

**Comment by sayakpaul** [comment]
You could first gather all the batch norm layers by looping through `model.layers`. Once you have done that, you can extract `gamma` by accessing it as an attribute of the layer. 

Refer to the [official documentation](https://keras.io/api/layers/normalization_layers/batch_normalization) of BatchNormalization in Keras for more details on the exact names of the attributes.
